.gitlab-ci.yml

```
@@ -649,7 +649,7 @@ run_dogstatsd_size_test:
     - inv -e dogstatsd.size-test --skip-build
 
 # check the size of the static dogstatsd binary
-run_dogstatsd_arm_size_test:
+dogstatsd_arm_size_test:
```

```suggestion
dogstatsd_arm64_size_test:
```

to be in line with the other arm64 jobs?